packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -273,6 +281,58 @@ You need to update your Authorization Server to use your custom domain to fix th
 3. Change the **Issuer** to use **Custom URL**.
 4. Try `./well-known/openid-configuration` again. It should now display your custom domain.
 
+## About configuring a custom email domain
+
+A custom email domain allows you to present a branded experience to your end users. Email that Okta sends to your end users appears to come from your custom email domain instead of `noreply@okta.com`. You can switch to a different custom domain or revert to the default Okta domain, but you can use only one email domain at a time.
+
+Okta sends your super admins a confirmation email after your custom domain is configured and operating correctly. To ensure continuous operation, Okta polls your custom email domain once every 24 hours. If a problem occurs, Okta alerts super admins by email, and Okta-generated emails are sent from the default domain `noreply@okta.com` until the problem is resolved.
+
+## Configure a custom email domain
+
+1. In the Admin Console, go to **Settings** and then **Emails & SMS**.
+
+2. Click the **Sender:** link (**Okta <noreply@okta.com>**) near the top of the page.
+
+3. On the Configure Email Sender dialog box, select **Custom email domain** as the type of sender that you want to send system notification emails from.
+
+4. In the **Email address to send from**, enter the email address that you want to send the system notification emails from. This is what displays in the emails sent to your users.
+
+5. Enter the **Name of sender**. This name appears as the sender in the emails sent to your users.
+
+6. In the **Mail domain to send from** box, enter a unique mail domain that your organization has dedicated for Okta to send mail from. Later in this procedure, you add the unique mail domain to the SPF record to your DNS zone (the root domain) as an include-statement to show that you allow Okta to send mail from this unique mail domain.
+
+7. Click the **Save & View Required DNS Records** button to save your changes and view your org's DNS records that you need to update before your settings can take effect.
+
+8. Update your DNS records using the provided values.
+
+9. When you have updated your DNS records through your domain provider, click **I've updated the DNS records**. Okta begins polling your DNS records until it detects your updates (may take up to 24 hours). Your configuration is pending until the DNS updates are detected.
```

Comment:
```suggestion
9. After you've updated your DNS records through your domain provider, click **I've updated the DNS records**. Okta begins polling your DNS records until it detects your updates (this may take up to 24 hours). Your configuration is pending until the DNS updates are detected.
```